core/src/banking_stage/unified_scheduler.rs

```diff
@@ -124,8 +139,16 @@ pub(crate) fn ensure_banking_stage_setup(
                     let (priority, _cost) =
                         calculate_priority_and_cost(&tx, &compute_budget_limits.into(), &bank);
                     let task_id = BankingStageHelper::new_task_id(task_id_base + i, priority);
+                    let consumed_block_size = packet.meta().size;
+                    let last_runnable_slot =
+                        last_slot_before_feature_activation.min(last_valid_slot_for_alt);
```

Comment:
This is correct given the current state...but I don't think it works well with how the tasks are unconditionally dropped after expiration. It needs to be re-tested/re-validated on the expiration.

Imagine I'm the first leader after the epoch boundary. The root bank I use will certainly be in the previous epoch; and that means I will have no transactions available for my block.

Since unified-scheduler also takes the votes, this would lead to an outage if we saw signficant adoption of unified-scheduler.